src/modules/auth/header.js

```diff
@@ -100,6 +100,8 @@ const mapStateToProps = state => {
 			username = userinfo.username;
 		}
 		else if (userinfo.preferred_username) {
```

Comment:
@Aringoaway You know what? I think it will be better to remove it completely. I think it has no use, since seacat auth service already offers `username` in several last versions. Sorry for misleading you. So, please remove everything from line 102 to 106, thanks